domains/certificates/Query.API/API/ContractService/ContractServiceImpl.cs

```diff
@@ -40,11 +40,6 @@ internal class ContractServiceImpl : IContractService
             return new GsrnNotFound();
         }
 
```

Comment:
We must have a check here if the meteringPoint can be used for issuing granular certificates. At the time of writing, Christoffer has an idea about the rules for a production metering point, but the rules for a consumption metering point is not defined.

Reply:
I'm unsure how to enforce the production scenario - I don't have AssetType on the materingPoint

Reply:
It is there, but I think it is difficult for us to fix right now. So I am resolving this